src/main/java/homeworks/andrea/secondAssignment/ParentClassOne.java

```diff
@@ -0,0 +1,39 @@
+package homeworks.andrea.secondAssignment;
+
+public abstract class ParentClassOne implements InterfaceABC {
```

Comment:
This class can contain the methods without the implementation, and his inheritors will inherit those behaviors